.github/workflows/build-image.yaml

```diff
@@ -1,109 +1,365 @@
-name: Build and push images
+name: Build and push multi-arch image
 
 on:
+  workflow_call:
+    inputs:
+      buildType:
+        description: Decide on what to build
+        required: true
+        type: string
+      gitRef:
+        required: false
+        type: string
+        default: ${{ github.sha }}
   workflow_dispatch:
     inputs:
       buildType:
```

Comment:
Previously the build types were in `build-config.yaml`, is that file now redundant?

Reply:
The file is still necessary to define the various app version and patch IDs. It also defines which apps are built by which "build types". So we do need to keep it.